secant/Features/SendFlow/SendFlowStore.swift

```diff
@@ -276,22 +279,38 @@ extension SendFlowViewStore {
 
     var bindingForConfirmation: Binding<Bool> {
         self.routeBinding.map(
-            extract: { $0 == .confirmation || self.bindingForSuccess.wrappedValue || self.bindingForFailure.wrappedValue },
+            extract: {
+                $0 == .confirmation ||
+                self.bindingForInProgress.wrappedValue ||
+                self.bindingForSuccess.wrappedValue ||
+                self.bindingForFailure.wrappedValue
+            },
             embed: { $0 ? SendFlowState.Route.confirmation : nil }
         )
     }
 
+    var bindingForInProgress: Binding<Bool> {
+        self.routeBinding.map(
+            extract: {
+                $0 == .inProgress ||
```

Comment:
This seems Pretty complicated and hard to test. If navigation depends on it I can foresee bugs from changes in other parts of the code.

Reply:
Responded in comment above.